alpine/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:13.2-alpine
```

Comment:
```suggestion
FROM postgres:13.4-alpine
```

Reply:
Rollbacked due to travis error as below
https://app.travis-ci.com/github/citusdata/docker/jobs/532229232